### ISSUE TITLE
[1.16.3] Exception in server tick loop

### ISSUE THREAD

**Issue by asura99** [bloop428/Excavation#23]
I can start the game normally, but it crashes when I try to destroy the grass.

This is crash report.
```
---- Minecraft Crash Report ----
// Shall we play a game?

Time: 10/29/20 10:57 AM
Description: Exception in server tick loop

java.lang.NoSuchMethodError: net.minecraft.world.IWorld.func_201672_e()Lnet/minecraft/world/World;
	at bloop.excavation.event.ServerEvent.veinMine(ServerEvent.java:29) ~[excavation:1.5.2] {re:classloading}
	at net.minecraftforge.eventbus.ASMEventHandler_1_ServerEvent_veinMine_BreakEvent.invoke(.dynamic) ~[?:?] {}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:85) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.3-service.jar:?] {}
	at net.minecraftforge.common.ForgeHooks.onBlockBreakEvent(ForgeHooks.java:561) ~[forge:?] {re:classloading}
	at net.minecraft.server.management.PlayerInteractionManager.func_180237_b(PlayerInteractionManager.java:230) ~[?:?] {re:classloading}
	at net.minecraft.server.management.PlayerInteractionManager.func_229860_a_(PlayerInteractionManager.java:220) ~[?:?] {re:classloading}
	at net.minecraft.server.management.PlayerInteractionManager.func_225416_a(PlayerInteractionManager.java:168) ~[?:?] {re:classloading}
	at net.minecraft.network.play.ServerPlayNetHandler.func_147345_a(ServerPlayNetHandler.java:860) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:fml:randompatches:RandomPatches ServerPlayNetHandler Transformer}
	at net.minecraft.network.play.client.CPlayerDiggingPacket.func_148833_a(SourceFile:40) ~[?:?] {re:classloading}
	at net.minecraft.network.play.client.CPlayerDiggingPacket.func_148833_a(SourceFile:10) ~[?:?] {re:classloading}
	at net.minecraft.network.PacketThreadUtil.lambda$checkThreadAndEnqueue$0(PacketThreadUtil.java:34) ~[?:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.util.concurrent.TickDelayedTask.run(SourceFile:18) ~[?:?] {re:classloading}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213166_h(ThreadTaskExecutor.java:172) ~[?:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.util.concurrent.RecursiveEventLoop.func_213166_h(SourceFile:23) ~[?:?] {re:computing_frames,re:classloading}
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:730) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_213166_h(MinecraftServer.java:156) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213168_p(ThreadTaskExecutor.java:134) ~[?:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.server.MinecraftServer.func_213205_aW(MinecraftServer.java:713) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_213168_p(MinecraftServer.java:707) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.concurrent.ThreadTaskExecutor.func_213160_bf(ThreadTaskExecutor.java:115) ~[?:?] {re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.server.MinecraftServer.func_213202_o(MinecraftServer.java:692) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.func_240802_v_(MinecraftServer.java:642) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.server.MinecraftServer.lambda$func_240784_a_$0(MinecraftServer.java:229) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_261] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.16.3
	Minecraft Version ID: 1.16.3
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_261, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1629488152 bytes (1554 MB) / 3814719488 bytes (3638 MB) up to 7635730432 bytes (7282 MB)
	CPUs: 8
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms1024m -Xmx8192m
	ModLauncher: 8.0.5+84+master.3310fe0
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-3.0.3-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.3-34.1.32-launcher.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.32-launcher.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.3-34.1.32-launcher.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.3-34.1.32-launcher.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/OptiFine_1.16.3_HD_U_G3.jar OptiFine TRANSFORMATIONSERVICE 
		/forge-1.16.3-34.1.32-launcher.jar fml TRANSFORMATIONSERVICE 
		/MixinBootstrap-1.0.5.jar mixinbootstrap TRANSFORMATIONSERVICE 
	FML: 34.1
	Forge: net.minecraftforge:34.1.32
	FML Language Providers: 
		javafml@34.1
		minecraft@1
	Mod List: 
		forge-1.16.3-34.1.32-client.jar                   |Minecraft                     |minecraft                     |1.16.3              |DONE      |NOSIGNATURE
		MouseTweaks-2.13-mc1.16.2.jar                     |Mouse Tweaks                  |mousetweaks                   |2.13                |DONE      |NOSIGNATURE
		jei-1.16.3-7.6.0.49.jar                           |Just Enough Items             |jei                           |7.5.0.43            |DONE      |NOSIGNATURE
		jeiintegration_1.16.3-6.0.0.8.jar                 |JEI Integration               |jeiintegration                |6.0.0.8             |DONE      |NOSIGNATURE
		excavation-1.5.2.jar                              |Excavation                    |excavation                    |1.5.2               |DONE      |NOSIGNATURE
		Patchouli-1.16.2-47.jar                           |Patchouli                     |patchouli                     |1.16.2-47           |DONE      |NOSIGNATURE
		create-mc1.16.3_v0.3+custom.jar                   |Create                        |create                        |mc1.16.3_v0.3+custom|DONE      |NOSIGNATURE
		Waystones_1.16.3-7.3.0.jar                        |Waystones                     |waystones                     |7.3.0               |DONE      |NOSIGNATURE
		Clumps-6.0.0.11.jar                               |Clumps                        |clumps                        |6.0.0.11            |DONE      |NOSIGNATURE
		FastWorkbench-1.16.3-4.4.0.jar                    |FastWorkbench                 |fastbench                     |4.4.0               |DONE      |NOSIGNATURE
		extlights-3.1.jar                                 |Extended Lights               |extlights                     |3.1                 |DONE      |NOSIGNATURE
		Placebo-1.16.3-4.3.3.jar                          |Placebo                       |placebo                       |4.3.3               |DONE      |NOSIGNATURE
		AutoRegLib-1.6-46.jar                             |AutoRegLib                    |autoreglib                    |1.6-46              |DONE      |NOSIGNATURE
		JustEnoughCharacters-1.16.3-4.1.2.jar             |Just Enough Characters        |jecharacters                  |4.1.2               |DONE      |NOSIGNATURE
		decorative_blocks-1.16.1-1.5.2.jar                |Decorative Blocks             |decorative_blocks             |1.5.2               |DONE      |NOSIGNATURE
		randompatches-1.16.2-1.22.1.2.jar                 |RandomPatches                 |randompatches                 |1.16.2-1.22.1.2     |DONE      |92:f6:29:d4:09:89:f5:f5:98:5e:20:34:31:d0:7b:58:22:06:bd:a5:d1:6a:92:6e:ac:3d:8d:18:c5:b2:5b:d7
		FastFurnace-1.16.3-4.3.0.jar                      |FastFurnace                   |fastfurnace                   |4.3.0               |DONE      |NOSIGNATURE
		Hwyla-forge-1.10.11-B78_1.16.2.jar                |Waila                         |waila                         |1.10.11-B78_1.16.2  |DONE      |NOSIGNATURE
		WAWLA-1.16.3-6.0.1.jar                            |WAWLA                         |wawla                         |6.0.1               |DONE      |NOSIGNATURE
		JustEnoughResources-1.16.3-0.12.0.100.jar         |Just Enough Resources         |jeresources                   |0.12.0.100          |DONE      |NOSIGNATURE
		VoxelMap-1.10.11_for_1.16.3.jar                   |VoxelMap                      |voxelmap                      |1.10.11             |DONE      |NOSIGNATURE
		forge-1.16.3-34.1.32-universal.jar                |Forge                         |forge                         |34.1.32             |DONE      |22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		Psi+1.16-87.jar                                   |Psi                           |psi                           |1.16-87             |DONE      |NOSIGNATURE
		Quark-r2.4-271.jar                                |Quark                         |quark                         |r2.4-271            |DONE      |NOSIGNATURE
		FastLeafDecay-v25.jar                             |FastLeafDecay                 |fastleafdecay                 |v25                 |DONE      |NOSIGNATURE
		chiselsandbits-0.2.2-ALPHA.jar                    |Chisels & bits                |chiselsandbits                |NONE                |DONE      |NOSIGNATURE
	Crash Report UUID: cabf4d64-cc7a-427a-b2a7-9813cb33fdb5
	[Psi] Active spell: None
	Patchouli open book context: n/a
	Player Count: 1 / 8; [ServerPlayerEntity['Lover_HZ'/166, l='ServerLevel[Survival Story]', x=-137.22, y=64.00, z=190.34]]
	Data Packs: vanilla, mod:mousetweaks, mod:jei, mod:jeiintegration, mod:excavation (incompatible), mod:patchouli (incompatible), mod:create (incompatible), mod:waystones (incompatible), mod:clumps (incompatible), mod:fastbench (incompatible), mod:extlights, mod:placebo (incompatible), mod:autoreglib (incompatible), mod:jecharacters (incompatible), mod:decorative_blocks, mod:randompatches (incompatible), mod:fastfurnace (incompatible), mod:waila, mod:wawla, mod:jeresources, mod:voxelmap (incompatible), mod:forge, mod:psi (incompatible), mod:quark (incompatible), mod:fastleafdecay (incompatible), mod:chiselsandbits (incompatible)
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'forge'
	OptiFine Version: OptiFine_1.16.3_HD_U_G3
	OptiFine Build: 20200916-082411
	Render Distance Chunks: 12
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 456.38
	OpenGlRenderer: GeForce GTX 1050 Ti/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8
```

**Comment by bloop428** [comment]
Mod is not yet compatible with 1.16.3. Finally decided to update it to 1.16.5. a year late i know

**Comment by bloop428** [comment]
Fixed by 799205f
